test.js

```diff
@@ -64,17 +64,8 @@ test('normalizeHttps option', t => {
 });
 
 test('normalizeHttps option with normalizeHttp', t => {
-	const opts = {normalizeHttps: true, normalizeHttp: true};
-
-	t.is(m('https://sindresorhus.com'), 'https://sindresorhus.com');
-	t.throws(() => {
-		m('http://sindresorhus.com', opts);
-	}, 'You should use one of normalizeHttp and normalizeHttps option.');
-	t.throws(() => {
-		m('https://www.sindresorhus.com', opts);
-	}, 'You should use one of normalizeHttp and normalizeHttps option.');
 	t.throws(() => {
-		m('//sindresorhus.com', opts);
+		m('https://www.sindresorhus.com', {normalizeHttps: true, normalizeHttp: true});
```

Comment:
You think my tests were not necessary @sindresorhus  ? Thats why you left just one?

Reply:
Yes. Whether it throws is not based on the input URL, so having multiple tests with different input URL changes nothing.

Reply:
Oh right, got you. thank for response!